src/main/java/no/nav/pensjon/selvbetjeningopptjening/logging/LoggingEndpoint.java

```diff
@@ -0,0 +1,30 @@
+package no.nav.pensjon.selvbetjeningopptjening.logging;
+
+import no.nav.security.token.support.core.api.Unprotected;
+import no.nav.pensjon.selvbetjeningopptjening.logging.LogMessage;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Unprotected
```

Comment:
Antar det er noe security filter som hindrer at hvem som helst kan kalle denne, men spør for sikkerhets skyld?

Reply:
Nei, det er ikke noe filter, så denne kan hvem som helst kalle.

Reply:
Hvem som helst burde vel ikke kunne logge ting?

Reply:
Enig i det.

Reply:
Bør fikses @sebassonav

Reply:
Så vil det fungere for alle use-cases med `@ProtectedWithClaims(issuer = ISSUER)`?

Reply:
https://github.com/navikt/pensjon-selvbetjening-opptjening-backend/pull/30